hazelcast/src/main/java/com/hazelcast/map/impl/LocalMapStatsProvider.java

```diff
@@ -176,6 +176,16 @@ private void addStatsOfNoDataIncludedMaps(Map statsPerMap) {
             String mapName = mapProxy.getName();
 
             if (mapConfig.isStatisticsEnabled() && !statsPerMap.containsKey(mapName)) {
+                // Lite members can invoke MapOperations, and their statistics are of importance for monitoring
+                // when Lite members are in use - so we should include them from our statsMap field in this case
+                if (nodeEngine.getLocalMember().isLiteMember()) {
+                    LocalMapStatsImpl localMapStats = statsMap.get(mapName);
+                    if (localMapStats != null && localMapStats.total() > 0) {
+                        statsPerMap.put(mapName, localMapStats);
+                        continue;
+                    }
+                }
+
                 statsPerMap.put(mapName, EMPTY_LOCAL_MAP_STATS);
```

Comment:
Could this be an alternative approach instead? 
```suggestion
	statsPerMap.put(mapName, mapProxy.getLocalMapStats());
```

And when destroying map for lite members, we also need to remove stats object after this change.
See `MapServiceContextImpl#destroyMap`

Advantage of this approach is we will not add a special case handling when creating stats for lite members.